<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogConfig</class>
 <widget class="QDialog" name="DialogConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>624</width>
    <height>729</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>系统配置</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>185</y>
     <width>211</width>
     <height>241</height>
    </rect>
   </property>
   <property name="title">
    <string>功能配置</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QCheckBox" name="chkCalib">
      <property name="text">
       <string>标定功能</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="chkCornFix">
      <property name="text">
       <string>角差修正</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="chkWave">
      <property name="text">
       <string>实时波形</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="chkWaveData">
      <property name="text">
       <string>历史波形</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="chkAnalog">
      <property name="text">
       <string>模拟量标定</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="chkTeds">
      <property name="text">
       <string>TEDS</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>13</y>
     <width>581</width>
     <height>162</height>
    </rect>
   </property>
   <property name="title">
    <string>外观配置</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>171</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>应用程序标题</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edtTitle">
    <property name="geometry">
     <rect>
      <x>192</x>
      <y>34</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbZh">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>132</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>中文</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbEn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>132</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>英文</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblIcon">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>80</y>
      <width>101</width>
      <height>71</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>110</y>
      <width>112</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>修改图标</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>185</y>
     <width>341</width>
     <height>251</height>
    </rect>
   </property>
   <property name="title">
    <string>系统功能</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QCheckBox" name="chkAdmin">
      <property name="text">
       <string>管理员模式</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="chkDebug">
      <property name="text">
       <string>调试模式</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="chkUartAuto">
      <property name="text">
       <string>串口自动扫描</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rbUart">
      <property name="text">
       <string>串口版本</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rbNet">
      <property name="text">
       <string>网络版本</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnOk">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>670</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>670</y>
     <width>131</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>581</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title">
    <string>隐藏参数</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QCheckBox" name="chkAlarm">
      <property name="text">
       <string>报警参数</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="chkSerial">
      <property name="text">
       <string>设备序列号</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="chkVersion">
      <property name="text">
       <string>版本号</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="chkChannel">
      <property name="text">
       <string>切换通道</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="chkView">
      <property name="text">
       <string>改变视图</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
